module-gui/gui/widgets/text/core/cursors/TextBlockCursor.cpp

```diff
@@ -326,6 +327,18 @@ namespace gui
         return false;
     }
 
+    auto BlockCursor::checkSentenceBeginningSings(const std::string &textToCompare) const -> bool
```

Comment:
```suggestion
    auto BlockCursor::checkSentenceBeginningSigns(const std::string &textToCompare) const -> bool
```

Reply:
and maybe add some tests for it? :P

Reply:
tests are added for that function with ". ".